pkg/rancher-components/types/index.d.ts

```diff
@@ -19,4 +19,4 @@ type ArrayListRow = {
 export type ArrayListData = {
     lastUpdateWasFromValue: boolean;
     rows: Array<ArrayListRow>,
-}
\ No newline at end of file
+}
```

Comment:
The lint rules in root require us to have an ending black newline at the end of each file.